Source/msgcmd.cpp

```diff
@@ -10,7 +10,8 @@
 // static float msgcmd_init_cpp_value = 0x7F800000;
 
 struct EXTERNMESSAGE {
-	LIST_LINK(EXTERNMESSAGE) m_Link;
+	LIST_LINK(EXTERNMESSAGE)
+	m_Link;

```

Comment:
Why split `m_Link` into second line from the type ?

Reply:
Because that's how clang_format is configured.